test/Microsoft.AspNetCore.SignalR.Redis.Tests/Docker.cs

```diff
@@ -70,6 +70,28 @@ private static string GetDockerLocation()
             return null;
         }
 
+        private void StartRedis(ILogger logger)
+        {
+            try
+            {
+                Run();
+            }
+            catch (Exception ex)
+            {
+                logger.LogError(ex, "Error starting redis docker container, retrying.");
```

Comment:
Wouldn't this be a possible infinite loop?

Do we want a counter and give up after N retries?

Reply:
It only calls `Run` twice, no looping

Reply:
Yeah, it's not recursive, it just looks kinda like it is because of the local func ;).

Add a delay? Just a second or two.

Reply:
Yeah, then don't :(

Reply:
Actually yeah, leave it in. It's OK here

Reply:
Cool yeah re-reading I see it now. :)